pkg/db/test/db.go

```diff
@@ -74,6 +75,13 @@ func GetDatabase(t *testing.T, inits ...DBInitializer) (name string, testDB *sql
 			return "", nil
 		}
 	}
+	t.Cleanup(func() {
```

Comment:
I didn't know about Cleanup

.... Also, looking at the rest of the docs, should we start marking these utility functions with `Helper()`

https://golang.org/pkg/testing/#T.Helper

> Helper marks the calling function as a test helper function. When printing file and line information, that function will be skipped. Helper may be called simultaneously from multiple goroutines.